archive/j/java/PrimeNumber.java

```diff
@@ -0,0 +1,31 @@
+public class PrimeNumber {
+
+	public static void main(String[] args) throws Exception {
+		try {
+			int n = Integer.parseInt(args[0]);
+			
+			if (n < 0)
```

Comment:
I'm not in love with the way this exception is handled. To me, it seems like it makes more sense to have some sort of branch that checks this initially before printing the error. That said, this passes all of our tests, so it's no big deal.

Reply:
I guess that I misunderstood you, what do you want is to check the parameter args[0] before assignment?